hello.c

```diff
@@ -0,0 +1,27 @@
+#include "common.h"
+//#include <cheri/cheric.h>
+#include <cheriintrin.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+
+/***
+ * This is my scratch file for testing things
```

Comment:
Out of date comment?

Reply:
Fixed in: https://github.com/capablevms/cheri-examples/pull/1/commits/1f8537c5e8e99633fb816022662eb0d728c14d84